hack/validate/vendor

```diff
@@ -13,6 +13,7 @@ validate_vendor_diff(){
 		vndr
 		# check if any files have changed
 		diffs="$(git status --porcelain -- vendor 2>/dev/null)"
```

Comment:
Should this check `vendor.conf` as well?

Reply:
vendor.conf is only updated manually, so that shouldn't be an issue